src/fmu/sumo/explorer/_child_object.py

```diff
@@ -1,5 +1,19 @@
+"""Contains child object classes"""
+
+from fmu.sumo.explorer._utils import get_surface
+
+
 class ChildObject:
```

Comment:
For future reference; perhaps we should move away from the Elasticsearch lingo and more towards FMU terms, to make this more familiar and intuitive to end user.

Reply:
Yes, I agree. But what terms shall be used then? Can we have a conversation about this, and resolve before feature freeze komodo next week?